### PR TITLE
User documentation branch

### PR DESCRIPTION
**Pull request by aeldesoky**
Added the markdown user documentation. help.js converts the markdown to html and then append it to the body in help.twig. Let me know if you want me to change anything.

This is related to #22 .

### CODE REVIEW

File: views/help.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      <div id="markdown" class="col">
+        {{ html | raw }}
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block scripts %}
+  {{ parent() }}
+  <script type="text/javascript" src="./routes/help.js"></script>
```
Comment (brandon1024) [inline views/help.twig]: I don't think you meant to serve the route file here hehe ;) You can remove this block, it's not needed :)

File: package.json [hunk 2]
```diff
@@ -13,13 +13,15 @@
     "express": "~4.15.5",
     "express-session": "^1.15.6",
     "font-awesome": "^4.7.0",
+    "fs": "0.0.1-security",
```
Comment (brandon1024) [inline package.json]: You don't need to pull this in as a dependency. `fs` is included with Node :)

File: views/help.twig (hunk 1 above)
Comment (brandon1024) [reply views/help.twig]: And could you move the stylesheet block above the body block? Just to keep consistent with the other twig files :)

File: public/stylesheets/help.sass [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.h1
```
Comment (brandon1024) [inline public/stylesheets/help.sass]: You can leave these out, they don't change anything :)

File: documents/help.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+## **About App Name** ##
+- - - -
+Oat cake dragée caramels dessert cotton candy gummi bears. Icing I love biscuit. Oat cake chocolate cake bonbon caramels sweet. Sesame snaps jelly beans pudding sweet dessert sugar plum sugar plum I love. Tiramisu chocolate cake marzipan pudding tiramisu oat cake icing I love. Cupcake bear claw dragée biscuit topping. Croissant I love gingerbread pie dessert wafer tart marshmallow brownie. I love toffee donut lollipop fruitcake donut sweet roll lollipop. Cotton candy bear claw cheesecake caramels dessert sesame snaps lollipop. Macaroon chocolate I love tiramisu bear claw marshmallow. Donut cake toffee biscuit wafer I love chocolate bar candy canes carrot cake. Macaroon marshmallow I love I love danish I love. Sweet dragée caramels toffee. Candy halvah I love pastry.
```
Comment (brandon1024) [inline documents/help.md:3]: LOL these are great 😆

**Review by brandon1024 — CHANGES_REQUESTED** [review]
Overall awesome! Just a couple of minor tweaks :)

**Comment by aeldesoky** [comment]
@brandon1024 Removed fs from dependencies, fixed help.twig, removed the stuff in help.sass cause it wasn't doing anything, and removed its link from help.twig. I am going to leave the sass file in case we decide later on that we want to style something specific in the documentation

**Review by brandon1024 — APPROVED** [review]

**Pull request merged**